server/src/main/java/keywhiz/service/resources/automation/v2/ClientResource.java

```diff
@@ -92,9 +94,16 @@
   @Consumes(APPLICATION_JSON)
   public Response createClient(@Auth AutomationClient automationClient,
       @Valid CreateClientRequestV2 request) {
+    return tagErrors(() -> doCreateClient(automationClient, request));
+  }
+
+  private Response doCreateClient(AutomationClient automationClient,
+      CreateClientRequestV2 request) {
     String creator = automationClient.getName();
     String client = request.name();
 
+    setTag("client", client);
+
     clientDAOReadWrite.getClientByName(client).ifPresent((c) -> {
```

Comment:
The most frequent errors we get for non-batch secret requests are for creating groups and clients.